cluster/k8s-multus-1.11.1/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubernetes 1.11.1 with multus network plugin in ephemeral containers
```

Comment:
Please reference this and OS README from docs/devel-guide.md. Also place there a general overview how to specific provider and number of nodes.